README.rst

```diff
@@ -34,6 +34,61 @@ One Time Setup
   # Here's how you might do that if you have virtualenvwrapper setup.
   mkvirtualenv -p python3.8 learning-assistant
 
+In your ``private.txt`` requirements file in edx-platform, add:
```

Comment:
I think we need to specify somewhere that this repository should be cloned into the `../src` directory relative to the user's `devstack` directory. This could be mentioned in the "One Time Setup" paragraph.

Reply:
Yes, good call! Added as a comment above the cloning instructions.